eZ/Publish/Core/Search/Common/FieldValueMapper/BooleanMapper.php

```diff
@@ -32,7 +32,7 @@ public function canMap(Field $field)
     }
 
     /**
-     * Map field value to a proper Elasticsearch representation.
+     * Map field value to a proper search engine representation.
```

Comment:
Updated in https://github.com/ezsystems/ezpublish-kernel/pull/1697/commits/820743b19040ae853418224d54f25b6e723ff01e.